common/thorhelper/persistent.hpp

```diff
@@ -41,6 +51,8 @@ interface IPersistentSelectNotify
     virtual bool notifySelected(ISocket *sock,unsigned selected, IPersistentHandler* handler, bool shouldClose) = 0;
 };
 
-IPersistentHandler* createPersistentHandler(IPersistentSelectNotify* notify, int maxIdleTime = DEFAULT_MAX_PERSISTENT_IDLE_TIME, int maxReqs = DEFAULT_MAX_PERSISTENT_REQUESTS, PersistentLogLevel loglevel=PersistentLogLevel::PLogMin, bool enableDoNotReuseList=false);
+#define HTTP_VER_OFFSET 5
```

Comment:
minor: could this be moved out of the header, and a helper function that uses it be exposed instead?

Reply:
@jakesmith removed HTTP_VER_OFFSET and added helper function httpVerOffset, please review again...